rapids-cmake/cython/init.cmake

```diff
@@ -38,6 +38,11 @@ Result Variables
 #]=======================================================================]
 macro(rapids_cython_init)
   list(APPEND CMAKE_MESSAGE_CONTEXT "rapids.cython.init")
+  include("${rapids-cmake-dir}/cmake/detail/policy.cmake")
+  rapids_cmake_policy(DEPRECATED_IN 24.02
+                      REMOVED_IN 24.08
```

Comment:
Removing in 24.06 would also be fine by me.

Reply:
I'm fine with either. @robertmaynard you can make the final call.

Reply:
Given that not all consumers of rapids are using the same release frequency I think we can wait to 24.08 to yank this